dataprovider/src/main/java/cm/aptoide/pt/dataprovider/model/v7/discovery/Video.java

```diff
@@ -0,0 +1,6 @@
+package cm.aptoide.pt.dataprovider.model.v7.discovery;
+
+
+public class Video {
+
+}
```

Comment:
We shouldn't forget that this is already created or else we will duplicated code - same for the GetVideos

Reply:
this is outside the scope of this ticket, it will be fixed in the api part;